base/file_name_utils.cpp

```diff
@@ -58,11 +58,7 @@ string GetDirectory(string const & name)
 
 string::value_type GetNativeSeparator()
 {
-#ifdef OMIM_OS_WINDOWS
```

Comment:
1. Why?
2. If it is not needed, then is the GetNativeSeparator function needed?